pkgs/os-specific/linux/sssd/default.nix

```diff
@@ -39,7 +41,11 @@ stdenv.mkDerivation {
       --without-semanage
       --with-xml-catalog-path=''${SGML_CATALOG_FILES%%:*}
       --with-ldb-lib-dir=$out/modules/ldb
-      --with-nscd=${glibc.bin}/sbin/nscd
+      ${lib.optionalString (nscdConfig != null) ''
+        --with-nscd=${glibc.bin}/sbin/nscd
+        --with-nscd-conf=${nscdConfig}
```

Comment:
Ugh, why does it need nscd config? We can symlink it into `/etc` to make it accessible. Otherwise people need to rebuild SSSD each time they change config, which is not nice.